astropy/time/setup_package.py

```diff
@@ -13,20 +12,12 @@
 SRC_FILES = [os.path.join(C_TIME_PKGDIR, filename)
              for filename in ['src/parse_times.c']]
 
-if sys.platform.startswith('win'):
```

Comment:
I removed it initially since I have not needed it for other ufuncs (like in erfa), but must admit I don't really know -- I'm essentially relying on CI here...